<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameResultWidget</class>
 <widget class="QWidget" name="GameResultWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>73</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,2">
     <item>
      <widget class="QLabel" name="laGame">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbPlayer1Score">
       <item>
        <property name="text">
         <string notr="true">Auto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true"/>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">29</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">28</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">27</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">26</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">25</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">24</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">23</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">22</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true"/>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">21</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true"/>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">19</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">18</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">17</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">14</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true"/>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbPlayer2Score">
       <item>
        <property name="text">
         <string notr="true">Auto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true"/>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">29</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">28</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">27</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">26</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">25</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">24</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">23</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">22</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true"/>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">21</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true"/>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">19</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">18</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">17</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">14</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true"/>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cbPlayer1Score</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>GameResultWidget</receiver>
   <slot>onScoreSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbPlayer2Score</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>GameResultWidget</receiver>
   <slot>onScoreSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onScoreSelectionChanged()</slot>
 </slots>
</ui>
